packages/integration/lib/src/widgets/lists/feed/feed_list.dart

```diff
@@ -527,6 +529,7 @@ class _LMFeedListState extends State<LMFeedList> {
                   LMFeedAnalyticsBloc.instance.add(
                       const LMFeedFireAnalyticsEvent(
                           eventName: LMFeedAnalyticsKeys.postCreationStarted,
+                          widgetSource: LMFeedWidgetSource.universalFeed,
```

Comment:
check this also, is it right to assign widgetSurce: LMFeedWidgetSource.universalFeed ?

Reply:
Have we cleared this with product team - to modify the keys being sent back within the analytic event?

Reply:
We aren't changing anything in the properties map defined by product team. 
Have aligned this change with @ishaan1607